<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenBuildConfigWidget</class>
 <widget class="QWidget" name="GenBuildConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="currentBuildSyslabel">
        <property name="text">
         <string>使用的构建系统：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="BuildSysComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>448</width>
        <height>569</height>
       </rect>
      </property>
      <widget class="QLabel" name="cmakeConfigLabel">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>141</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>CMake配置</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>406</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="cmakePathLabel">
          <property name="text">
           <string>cmake路径：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="cmakePathLineEdit"/>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="cmakePathBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="cmakelistFileLabel">
          <property name="text">
           <string>CMakeLists文件路径：：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="cmakelistFileLineEdit"/>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="cmakelistFileBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="cmakeCComplierLabel">
          <property name="text">
           <string>c编译器路径：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="cmakeCComplierLineEdit"/>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="cmakeCComplierBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="cmakeCXXComplierlabel">
          <property name="text">
           <string>c++编译器路径：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="cmakeCXXComplierLineEdit"/>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="cmakeCXXComplierBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>使用的构建系统：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="cmakeUseBuildComboBox"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>构建系统路径：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="cmakeUseBuildEdit"/>
        </item>
        <item row="6" column="2">
         <widget class="QToolButton" name="cmakeUseBuildBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>build目录：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="cmakeBuildDirlineEdit"/>
        </item>
        <item row="7" column="2">
         <widget class="QToolButton" name="cmakeBuildDirBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>install目录：</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="cmakeInstallLineEdit"/>
        </item>
        <item row="8" column="2">
         <widget class="QToolButton" name="cmakeInstallBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>360</y>
         <width>111</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>makefile配置</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>354</width>
         <height>79</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>c编译器路径：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="mkCComplierLineEdit"/>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="mkCComplierBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>c++编译器路径：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="mkCXXComplierLineEdit"/>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="mkCXXComplierBtn">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
